src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -28,6 +28,25 @@ typedef NS_ENUM(NSInteger, MTRCommissioningStatus) {
     = 3,
 } API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * Read from controllee's Information  at Commissioning
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRReadCommissioningInfo : NSObject
+
+/**
+ * Controllee's VendorID Attribute of Basic Information cluster
```

Comment:
Again, I would not document here where the information comes from.  For example, maybe we'll want to create these objects during device attestation later on, when we have this information from cryptographically signed certificates.

Reply:
The contents of this part are specified in the document below.
Is there anything additional that needs to be done?